src/tests/nurse/test_views.py

```diff
@@ -219,10 +224,17 @@ def test_send_email_to_doctor_no_patient(self, client):
             end_date="2022-07-31",
             patient=None,
         )
-        response = client.post(self.url)
+        response = client.post(self.url, self.valid_payload)
         assert response.status_code == status.HTTP_404_NOT_FOUND
         assert response.data == {"error": "This prescription has no patient"}
 
+    def test_send_email_to_doctor_invalid_additional_info(
+        self, client, prescription, user
+    ):
+        # Test with invalid additional_info
+        response = client.post(self.url, self.invalid_payload)
+        assert response.status_code == status.HTTP_400_BAD_REQUEST
+
```

Comment:
Assert the response.data  error dict just like the other tests above because we want to make sure it's failing on this field. Also you can remove the obvious comment, it's a 2 liner test with the description in the method name already

Reply:
Add assert and remove comment